f2/src/BusinessPage.js

```diff
@@ -18,6 +18,7 @@ export const BusinessPage = () => {
   const whatWasAffectedNavState = state.whatWasAffectedOptions
 
   whatWasAffectedNavState.currentPage = whatWasAffectedPages.BUSINESS
+  navigate(doneForms, whatWasAffectedNavState)
```

Comment:
If we are going to change where this is called we should change the name, `navigate` might cause some confusion. Perhaps `updateNavigation` would better describe the function.

Reply:
Changed.